src/main/java/mate/controller/CarsController.java

```diff
@@ -0,0 +1,24 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/car/show")
```

Comment:
check the [endpoints style convention](https://mate-academy.github.io/jv-program-common-mistakes/java-web/web-practice/java-web-practice), fix in all controllers
```suggestion
@WebServlet(urlPatterns = "/cars/all")
```